scripts/generateTS.sh

```diff
@@ -232,7 +249,48 @@ export {
 export {
   HttpsApiDataplatformIbmComSchemasCommonCanvasPalettePaletteV3SchemaJson as PipelineFlowPalette,
   CategoryDef
-} from \"./palette-v3.ts\";"
+} from \"./palette-v3.ts\";
+export {
+  PropertyDefinitionsSchema,
+  ConditionsDefinition,
+  MessageDefinition,
+  EvaluateDefinition,
+  ValidationDefinition,
+  FailMessageDefinition,
+  AndDefinition,
+  OrDefinition,
+  ConditionDefinition,
+  FilterConditionDefinition,
+  EnabledDefinition,
+  VisibleDefinition,
+  FilterDefinition,
+  EnumFilterDefinition,
+  AllowChangeDefinition,
+  DefaultValueDefinition,
+  ParameterDefinition as OperatorParameterDefinition,
+  ParameterRefDefinition,
+  ComplexTypeDefinition as OperatorComplexTypeDefinition,
+  WDPUIHintsSchema,
```

Comment:
How do we come up with the naming here?  WDP looks odd.

Reply:
This is the title without spaces - in https://github.com/elyra-ai/pipeline-schemas/blob/main/common-pipeline/operators/uihints-v3-schema.json#L5

I will change the title to `UI Hints Schema`